pkg/nvml/mock/dgxa100/dgxa100.go

```diff
@@ -26,325 +26,49 @@ import (
 
 type Server struct {
 	mock.Interface
-	Devices [8]nvml.Device
+	Devices           [8]nvml.Device
+	DriverVersion     string
+	NvmlVersion       string
```

Comment:
Question: When is the `NvmlVersion` relevant?

Reply:
Not sure exactly, but the code already had an API call for it, so I updated it to pull from the Server field now instead of hard-coding it in the method call. I assume mig-parted must rely on it since that is where this mock server was originally housed.